app/scripts/app.js

```diff
@@ -53,14 +54,19 @@ app.config(function($routeProvider) {
       templateUrl: 'views/confirm-order.html',
       controller: 'OrderController'
     })
+    .when('/payment/:id', {
+      templateUrl: 'views/payment.html',
+      controller: 'PaymentController'
+    })
     .otherwise({
       redirectTo: '/'
     });
 });
 
 app.constant('CONFIG', {
-  // 'BASE_URI': '//localhost:3000',
-  'BASE_URI': '//tma-develop.herokuapp.com',
+  'BASE_URI': '//localhost:3000',
+  // 'BASE_URI': '//tma-develop.herokuapp.com',
+  STRIPE_PUBLISHABLE_KEY: ""
```

Comment:
other option would be to send this `STRIPE_PUBLISHABLE_KEY` from backend in every `GET order/:id`
something like:
`json.stripe_publishabe_key STRIPE_PUBLISHABLE_KEY if (order.status == "pending")`
and define `STRIPE_PUBLISHABLE_KEY` on backend in `application.yml` tohether with `STRIPE_API_KEY`